### PR TITLE
Update doc links to new urls

### PR DESCRIPTION
**Pull request by payzer**
This change updates the documentation links to point at the new URLs that contain the most recent documentation.

### CODE REVIEW

**Comment by MSFT-Vince** [comment]
Quick question about the AKA links: I see that they resolve to valid URLs, so is the change necessary here because the landing pages are no longer relevant for the scope of this doc?

Alternatively, why wouldn't we change where the AKA links are pointing?

**Comment by payzer** [comment]
Great suggestion. You are right. We should change the aka.ms links. At least the one for the Unity guide. I'll work on that.

**Review by danapayne — APPROVED** [review]

File: Package/Assets/GDK-Tools/Source/Editor/GdkPlugin.cs [hunk 1]
```diff
@@ -13,13 +13,13 @@ public static class GdkPlugin
...
         {
-            Application.OpenURL("https://docs.microsoft.com/gaming/gdk/");
+            Application.OpenURL("https://aka.ms/gamedevdocs");
         }
 
         [MenuItem("GDK/Documentation/Unity Integration Documentation (GDK)")]
         private static void OpenUnityDocs()
         {
-            Application.OpenURL("https://docs.microsoft.com/gaming/gdk/_content/gc/get-started-with-pc-dev/get-started-with-unity-pc/gdk-unity-end-to-end-guide");
+            Application.OpenURL("http://aka.ms/gdk_unity");
```
Comment (MSFT-Vince) [inline Package/Assets/GDK-Tools/Source/Editor/GdkPlugin.cs]: Would recommend updating to:

Application.OpenURL("https://aka.ms/gdk_unity");
```suggestion
            Application.OpenURL("https://aka.ms/gdk_unity");
```

Comment (payzer) [reply Package/Assets/GDK-Tools/Source/Editor/GdkPlugin.cs]: Good idea. I'll do that.

**Pull request merged**